style.css

```diff
@@ -0,0 +1,78 @@
+
+.container {
+    color: white;
+    margin: 0 auto;
+}
+
+header {
+  background-color: #ebebeb;
+  padding: 30px;
+  text-align: center;
+  font-size: 35px;
+  color: white;
+  border-style: solid;
+}
+
+Nav {
+    background-color: #ebebeb;
+    padding: 30px;
+    text-align: center;
+    font-size: 35px;
+    color:white;
+    border-style: solid;
+}
+
+body {
+font-family: Arial, Helvetica Neue, Helvetica, sans-serif;
+color: #777;
+background-color: #777;
+width: 960px;
+font-size: 18px;
+margin: 0px;
+}
+
+Section {
+    background-color: #ebebeb;
+    width: 495px;  
+    color: white;
+    float: left; 
+    border-style: solid; 
+    height: 300px;
+}
+
+Aside {
+    background-color: #ebebeb;
+    color: white;
+    float: right;
+    border-style: solid;
+    width: 47%;
+    height: 300px;
+}
+
+
+Article {
+    color: #ebebeb;
+    background-color: #777;
+    width: 400px;
+    height: 150px;
+    }
+    
+    h1 {
+        color: #777;
+        background-color: #ebebeb;
+    }
```

Comment:
Make sure your code is formatted and easily readable! It is very easy to make mistakes and miss things because of bad coding format.